tests/test_smartcar.py

```diff
@@ -6,382 +6,475 @@
+            {
+                "response_type": "code",
+                "client_id": self.client_id,
+                "redirect_uri": self.redirect_uri,
+                "approval_prompt": "force",
+                "state": "stuff",
+                "scope": " ".join(self.scope),
+                "single_select": False,
+            }
+        )
 
         expected_params = parse_qs(expected_query)
         actual_params = parse_qs(actual_query)
 
         assertDeepEquals(self, expected_params, actual_params)
 
     def test_get_auth_url_flags_country(self):
-        client = smartcar.AuthClient(self.client_id, self.client_secret,
-                self.redirect_uri, self.scope)
-        actual = client.get_auth_url(force=True, state='stuff', flags=['country:DE'])
-        query = urlencode({
-            'response_type': 'code',
-            'client_id': self.client_id,
-            'redirect_uri': self.redirect_uri,
-            'approval_prompt': 'force',
-            'scope': ' '.join(self.scope),
-            'state': 'stuff',
-            'flags': 'country:DE'
-        })
-        expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
+        client = smartcar.AuthClient(
+            self.client_id, self.client_secret, self.redirect_uri, self.scope
+        )
+        actual = client.get_auth_url(force=True, state="stuff", flags=["country:DE"])
+        query = urlencode(
+            {
+                "response_type": "code",
+                "client_id": self.client_id,
+                "redirect_uri": self.redirect_uri,
+                "approval_prompt": "force",
+                "scope": " ".join(self.scope),
+                "state": "stuff",
+                "flags": "country:DE",
+            }
+        )
+        expected = smartcar.const.CONNECT_URL + "/oauth/authorize?" + query
 
         expected_params = parse_qs(expected)
         actual_params = parse_qs(actual)
 
         assertDeepEquals(self, expected_params, actual_params)
 
-
     @responses.activate
     def test_exchange_code(self):
         body = {
-            'grant_type': 'authorization_code',
-            'code': 'code',
-            'redirect_uri': self.redirect_uri
+            "grant_type": "authorization_code",
+            "code": "code",
+            "redirect_uri": self.redirect_uri,
         }
-        responses.add('POST', smartcar.const.AUTH_URL, json=self.expected)
-        actual = self.client.exchange_code('code')
-        self.assertIn('key', actual)
-        self.assertTrue(actual['expiration'] > datetime.utcnow())
-        self.assertTrue(actual['refresh_expiration'] > datetime.utcnow())
-        self.assertEqual(request().headers['Authorization'], self.basic_auth)
-        self.assertEqual(request().headers['Content-Type'], 'application/x-www-form-urlencoded')
+        responses.add("POST", smartcar.const.AUTH_URL, json=self.expected)
+        actual = self.client.exchange_code("code")
+        self.assertIn("key", actual)
```

Comment:
just realized that these use `camelCase` I would have expected it to be `snake_case`

Reply:
yeah same, the `setUp` method also is snake case oddly